beta/src/pages/learn/index.md

```diff
@@ -4,716 +4,486 @@ title: Quick Start
-
-If you're curious which products you use everyday are built with React, you can install the [React Developer Tools](/learn/react-developer-tools). Whenever you visit an app or a website built with React (like this one!), its icon will light up in the toolbar.
-
-### React uses JavaScript {/*react-uses-javascript*/}
-
-With React, you will describe your visual logic in JavaScript. This takes some practice. If you're learning JavaScript and React at the same time, you're not alone--but at times, it will be a little bit more challenging! On the upside, **much of learning React is really learning JavaScript,** which means you will take your learnings far beyond React.
-
-Check your knowledge level with [this JavaScript overview](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript). It will take you between 30 minutes and an hour but you will feel more confident learning React. [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) and [javascript.info](https://javascript.info/) are two great resources to use as a reference.
-
-<DeepDive title="Installation (optional)">
-
-If you're just starting to learn React, you don't need to install anything. Instead, we recommend to follow along using the interactive sandboxes that appear throughout this site. They look like this:
+Have a look at the result:
 
 <Sandpack>
 
 ```js
-function Greeting({ name }) {
-  return <h1>Hello, {name}</h1>;
+function MyButton() {
+  return (
+    <button>
+      Click me
+    </button>
+  );
 }
 
-export default function App() {
-  return <Greeting name="world" />
+export default function MyApp() {
+  return (
+    <div>
+      <h1>Welcome to my app</h1>
+      <MyButton />
+    </div>
+  );
 }
 ```
 
 </Sandpack>
 
-When you're ready to start a project, there are several options. You can write React code online and save your work in an environment like [CodeSandbox](https://react.new/). You can add React as a `<script>` tag to any HTML file to try it for a small part of your existing webpage. Or you can create a whole new React app, with or without a framework. **Use the [Installation](/learn/installation) page to pick what works best for you--but you can skip it for now.**
-
-</DeepDive>
+The `export default` keywords specify the main component in the file. If you're not familiar with some piece of JavaScript syntax, [MDN](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export) and [javascript.info](https://javascript.info/import-export) have great references.
 
-## Learn React {/*learn-react*/}
+## Writing markup with JSX {/*writing-markup-with-jsx*/}
 
-There are a few ways to get started:
+The markup syntax you've seen above is called JSX. It is optional, but most React projects use JSX for its convenience. Browsers don't understand JSX natively, but you can [set up a tranform locally](/learn/installation) when you're ready.
```

Comment:
Typo:

```suggestion
The markup syntax you've seen above is called JSX. It is optional, but most React projects use JSX for its convenience. Browsers don't understand JSX natively, but you can [set up a transform locally](/learn/installation) when you're ready.
```

Also not sure if people reading this are going to know what "transform" means in this context. 

Would a different phrasing work here?

> , so JSX is [typically transformed into plain JS by a project build tool]() so that browsers can run that code.